src/Provider/Provider.Web/Filters/CheckProviderBlockedFilter.cs

```diff
@@ -28,7 +28,7 @@ public CheckProviderBlockedFilter(IConfiguration configuration)
 
                 var ukprnFromUrl = context.RouteData.Values[RouteValues.Ukprn].ToString().ToUpper();
 
-                if (blockedProviderAccountIds.Contains(ukprnFromUrl))
+                if (!blockedProviderAccountIds.Contains(ukprnFromUrl))
```

Comment:
Now this is a whitelist can we rename this to `whitelistedProviderAccountIds`

Reply:
they are not account id's, they are ukprns